README.md

```diff
@@ -20,11 +20,11 @@ Bug reports and feature requests can be submitted [here](https://github.com/Ante
 We also hold regular community calls to discuss anything AntennaPod-related. [Come join the next call](https://forum.antennapod.org/t/monthly-community-call/1869)!
 
 ## Help to test AntennaPod
-AntennaPod has many users and we don't want them to run into trouble when we add a new feature. It's important that we have a significant group test our app, so that we know all possible combinations of phones, Android versions and use cases work as expected. Check out our wiki on how to join our [Beta testing program](https://antennapod.org/documentation/general/beta)! If a bug is reported during the beta period, chances are high that it will be fixed before the stable version. If it is reported later, fixing might take another full beta cycle. So definitely let us know if something is not right.
+AntennaPod has many users and we don't want them to run into trouble when we add a new feature. It's important that we have a significant group test our app, so that we know all possible combinations of phones, Android versions and use cases work as expected. Check out our wiki on how to join our [Beta testing program](https://antennapod.org/documentation/general/beta)! If a bug is reported during the beta period, chances are high that it will be fixed before the next stable version. If it is reported later, fixing might take another full beta cycle. So definitely let us know if something is not right.
```

Comment:
I'm not a native English speaker but to me this sounds like `next` does not refer to the stable version corresponding to that beta but to the next stable version after that.

Reply:
"Next", when used as an English adjective, means "nearest in space or position", so it works here, but I may have even better English words. What do you think of "upcoming"?  Another good choice is "subsequent".  Just let me know your preference... no problem to change it to your liking.

BTW, thank you for all your kind efforts on this project.  I always look forward to reading the changelog to see what you and the others have improved.  This project makes my life a little better, so I am grateful to you all.  Also thank you for the smile on my face every time I read your pseudonym.  🐹 :)

Reply:
What about 'first next'?

Also, I just noticed that the beta sentence refers to 'wiki' but this should be 'documentation'. Would be great if that could be corrected while we're at it.

Reply:
I think I like "upcoming" most